src/plugins/visualizations/public/plugin.ts

```diff
@@ -182,6 +182,14 @@ export class VisualizationsPlugin
     { data, expressions, uiActions, embeddable, dashboard }: VisualizationsStartDeps
   ): VisualizationsStart {
     const types = this.types.start();
+    const savedAugmentVisLoader = createSavedAugmentVisLoader({
```

Comment:
We can import this from vis_augmenter like here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/7f553f30f7c9f644da9e663d6a7274769ddb65cf/src/plugins/visualizations/public/plugin.ts#L202

Reply:
So here we actually can't since we don't take a `requiredPlugins` dependency on `visAugmenter`. We do still get the `createSavedAugmentVisLoader` fn available since it is a `requiredBundle`. We can't add as a `requiredPlugin` since it would lead to circular dependency between the 2 plugins.